config/purposes/lcmb.yml

```diff
@@ -0,0 +1,45 @@
+---
+LCMB Cherrypick:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: true
+  :input_plate: true
+  :presenter_class: Presenters::StockPlatePresenter
+LCMB End Prep:
+  :asset_type: plate
+LCMB Lib PCR:
+  :asset_type: plate
+  :presenter_class: Presenters::PcrPresenter
+  :creator_class: LabwareCreators::TaggedPlate
+  # TODO: do we need this limitation (from WGS)?
+  # warnings:
+  #   pcr_cycles_not_in:
+  #     - 6
+  :tag_layout_templates:
+    - pWGS UDI tag layout v2 A
+    - pWGS UDI tag layout v2 B
+    - pWGS UDI tag layout v2 C
+    - pWGS UDI tag layout v2 D
+LCMB Lib PCR-XP:
+  :asset_type: plate
+  :default_printer_type: :plate_b
```

Comment:
Assume this printer_type and label_template already exist? Their names just looks a bit strange!

Reply:
Yes they do!